### PR TITLE
Add abbreviated collection centre to day overview PDF

### PR DESCRIPTION
**Pull request by mbarton**
Also removes one of the checkboxes to help make room for it
Screenshot 2021-11-12 20 40 24

### CODE REVIEW

**Comment by dan-langridge** [comment]
Looks great. I think just showing collection abrv makes sense.

Can you shrink the checkbox column to fit the size of the checkbox? No worries if not though, happy to go live with it 👍

**Comment by mbarton** [comment]
@dan-langridge ah unfortunately the columns are fixed width and not fluid to their contents. I have to keep that space there for the red flag if set

**Comment by dan-langridge** [comment]
@mbarton oh yeah, of course, I forgot about the red flag! Thanks then, looks great'

File: backend/foodbank_southlondon/templates/day-overview.html [hunk 1]
```diff
@@ -130,6 +129,7 @@ <h1>DAY OVERVIEW</h1>
                         </span>
                     </td>
                     <td>{{ request['time_of_day'] }}</td>
+                    <td>{{ collection_centres[request['collection_centre']]['abbr'] if request['shipping_method'] == 'Collection' else '' }}</td>
```
Comment (adamcunnington) [inline backend/foodbank_southlondon/templates/day-overview.html:132]: I think that the `else ''` might actually be extraneous

**Comment by adamcunnington** [comment]
@dan-langridge happy for this to be merged?

**Comment by dan-langridge** [comment]
Yep happy to merge!

**Review by adamcunnington — APPROVED** [review]

**Pull request merged**